test/greed/score_test.rb

```diff
@@ -3,53 +3,71 @@
 require 'test/unit'
 require 'pry'
 
-include Greed
-# Tests from Ruby Koans "about_scoring_project"
-class TestScore < Test::Unit::TestCase
+module Greed
+  # Tests from Ruby Koans "about_scoring_project"
+  class TestScore < Test::Unit::TestCase
+    $score = Score.new
```

Comment:
Naming this `$score` stores a new score as a global variable. Global variables should never be used.

In tests specifically, it's desirable to isolate the tests as much as possible. Running one test should not affect the other tests in any way. Since these are using the same `Score` object there is a danger of that happening.

Instead, maybe look into the `Test::Unit` `setup` method, or just call `Score.new` in each test.